_tests/integration/version_test.go

```diff
@@ -14,7 +14,7 @@ func Test_VersionOutput(t *testing.T) {
 	{
 		out, err := command.RunCommandAndReturnCombinedStdoutAndStderr(binPath(), "version")
 		require.NoError(t, err, out)
-		require.Equal(t, "0.13.2", out)
+		require.Equal(t, "0.0.0-development", out)
```

Comment:
shouldn't it be `0.13.2` ?

Reply:
The final version number gets injected at build time based on the git tag of the release.